contrib/babelfishpg_tsql/sql/collation.sql

```diff
@@ -201,3 +201,8 @@ INSERT INTO sys.babelfish_helpcollation VALUES (N'vietnamese_cs_as', N'Vietnames
 INSERT INTO sys.babelfish_helpcollation VALUES (N'vietnamese_ci_as', N'Vietnamese, case-insensitive, accent-sensitive, kanatype-insensitive, width-insensitive');
 INSERT INTO sys.babelfish_helpcollation VALUES (N'vietnamese_ci_ai', N'Vietnamese, case-insensitive, accent-insensitive, kanatype-insensitive, width-insensitive');
 
+create or replace function sys.remove_accents(IN sys.NVARCHAR) RETURNS sys.NVARCHAR
```

Comment:
Seems like this function wont be used externally, Can we name this with "_internal" suffix?

Reply:
Updated